### DISCUSSION TITLE
How to generate a report to show the difference only

### DISCUSSION THREAD

**Discussion by jundoerr** [WinMerge/winmerge#2148] [General]
Current reports will have every line of the data in both files. Can I get a report that shows only the difference?

thank you, Jun

**Comment by sdottaka** [comment]
The appearance of the report is currently designed to resemble the appearance of the current window. To generate the expected report showing only differences, please select the "View → Diff Context → 0 Lines" menu item, and then output the report.

**Reply by renezito** [reply]
wow... thanks for this question and reply !!!   Excellent tool !   I think this question must be moved to closed or some similar status.

**Comment by phreeplot** [comment]
It does seem to remember the 'Diff context' that you set between invocations, which is good. It would be nice to be able to set the number of context lines with a command line option.

**Comment by sdottaka** [comment]
https://github.com/WinMerge/winmerge/discussions/2117 This is an example of specifying Diff Context on the command line.

**Comment by phreeplot** [comment]
Yes, thanks. I've got it all working now. Great app.
